src/mscorlib/src/System/Text/ASCIIEncoding.cs

```diff
@@ -78,12 +80,157 @@ public override unsafe int GetByteCount(char* chars, int count)
             return EncodingForwarder.GetByteCount(this, chars, count);
         }
 
-        public override int GetBytes(String chars, int charIndex, int charCount,
-                                              byte[] bytes, int byteIndex)
+        public unsafe override byte[] GetBytes(String s)
+        {
```

Comment:
Ah, full framework all go via the internal virtual `GetBytes(char* chars, int charCount, byte* bytes, int byteCount, EncoderNLS encoder)` so is only adding the `GetBytes(String s)` overload